@here/harp-mapview/lib/text/TextElementState.ts

```diff
@@ -80,15 +87,18 @@ export class TextElementState {
      *
      * @returns [[AnchorPlacement]] object containing vertical/horizontal align.
      */
-    get textPlacement(): AnchorPlacement {
+    get textPlacement(): TextPlacement {
```

Comment:
Method documentation needs to be updated.

Reply:
Yep, of course.